pkg/handler/pullrequest/handler.go

```diff
@@ -75,7 +75,7 @@ func openOrSync(gitRepo *git.Git, pr *github.PullRequestPayload, gh ghclient.GH)
 	if config != nil && config.LabelApproved != nil {
 		readyToReviewMsg += fmt.Sprintf("\n🚀 Full E2E won't run until the %q label is applied. "+
 			"I will add it automatically once the PR has %d approvals, or you can add it manually.",
-			*config.LabelApproved.Label, *config.LabelApproved.Approvals)
+			*(config.LabelApproved.Label), *(config.LabelApproved.Approvals))
```

Comment:
This makes no sense TBH.

From what I can tell, the address operator has less precedence than selector in go according to the specs.

Also you can see for the label it actually works as intended:
>  rocket Full E2E won't run until the "ready-to-test" label is applied. I will add it automatically once the PR has 824671612280 approvals, or you can add it manually.

Also, the actual value of approvals is used correctly in the handler:
https://github.com/submariner-io/submariner-bot/blob/devel/pkg/handler/pullrequestreview.go#L54

Why would it work as expected in `pullrequestreview.go` but not here?

Reply:
https://golang.org/ref/spec#Selectors

>  As an exception, if the type of x is a defined pointer type and (*x).f is a valid selector expression denoting a field (but not a method), x.f is shorthand for (*x).f. 

Even following the examples, if you'd want to dereference `config` you'd need to write `(*config).LabelApproved` not `*config.LabelApproved`

Reply:
I see, then I can't understand why it's happening.

I will go back to printing the parsed (struct) and see what we get as debug